### PR TITLE
Fixes Oversight With Loadout in #16132

### PR DESCRIPTION
**Pull request by Leanfrog**
## What Does This PR Do
This PR removes the ability for Pod Pilots to choose the security bomber jacket in the loadout as they already start with it, oversight on my part, whoops

## Why It's Good For The Game
Pod pilots don't need to start with an extra copy of their jacket they already start with

## Changelog
:cl:
tweak: Sec Pod Pilots can no longer receive an extra sec bomber jacket if it is selected in loadout
/:cl:

### CODE REVIEW

**Review by SabreML — APPROVED** [review]

**Review by dearmochi — APPROVED** [review]

**Comment by AffectedArc07** [comment]
Honestly I dislike this.

There's lots of jobs where you can select something in the loadout that you already get. This is just an exclusion for the sake of it.

Also keep in mind that there's only one of these, it's on their table not their bag, so a second pod pilot spawning after the first won't have one.



TLDR: Not in favour

**Comment by Spacemanspark** [comment]
I don't particularly see the harm in the pod pilot being able to have a second one tbh. Hell, one could argue that it makes sense for people to carry an extra pair of clothes around, just in case.

**Comment by Wojz** [comment]
So you're telling me I can start with THREE bomber jackets?!

**Comment by variableundefined** [comment]
Objection for same reason as Affected. Closing.
